### ISSUE TITLE
Backfill RX

### ISSUE THREAD

**Issue by sandragall** [LoopKit/Loop#1337]
buildDateString: Wed Jan 15 22:45:28 CST 2020
* xcodeVersion: 11C504

## DeviceDataManager
* launchDate: 2020-06-20 03:35:35 +0000
* lastError: Optional((date: 2020-06-19 19:32:19 +0000, error: GlucoseBackfillRxMessage time interval not reflected in glucose: 4953386 - 4982480, buffer: 1349011551 - 3199495824))
* lastBLEDrivenUpdate: 20

**Comment by summerrosek** [comment]
I'm not sure if this is helpful at all, I am not very knowledgeable with troubleshooting from looking at the issue logs, but I had the same error message yesterday off and on.  Is there something in particular happening to your loop readings when you get this message?  

My Loop app, and initially, my apple watch, were both reading a much higher BG than my Dexcom G6 app.  A finger stick proved Dexcom was accurate.  But my loop was green and claimed everything was right with no errors.  I did not get the "GlucoseBackfillRxMessage time interval not reflected in glucose" error until about an hour later.  

Earlier in the day I had accidentally switched my date and time from automatic and it was running at a different time zone for about an hour.  I did not think anything of it until after (hours) of troubleshooting, I looked at the logs of my BG data in the Health app;  I had some logs that "went back" in time because of the time change, making two logs of the same date and time.  This was messing with my loop app reading accurately.  I deleted those logs in the health app and restarted loop.  This fixed the issue and I haven't had a similar occurrence since. 

My watch was an issue that happened to compound at the same time: I allowed the new watchOS update and it isn't entirely compatible with Dexcom.  When I was troubleshooting the loop/dexcom issue mentioned above, I deleted and reinstalled Dexcom on my phone, but it would spin endlessly and fail to install on my watch.  Thankfully I had a back up of my watch from the previous version and I was able to re-pair it and it installed and is syncing with Loop.
